website/README.md

```diff
@@ -2,32 +2,31 @@
 ---
 
 - [Prerequisites](#prerequisites)
-  - [Docker](#docker)
   - [OSX, Linux, Windows](#osx-linux-windows)
+  - [Docker](#docker)
+  
 - [Setting up the development environment](#setting-up-the-development-environment)
   - [Clone the repository](#clone-the-repository)
   - [OSX](#osx)
   - [Linux](#linux)
   - [Windows](#windows)
 - [Running the stack](#running-the-stack)
   - [Create the .envrc and .env files](#create-the-envrc-and-env-files)
-  - [Docker](#docker-1)
+  - [Running the website](#running-the-website-application)
   - [OSX, Linux, and Windows](#osx-linux-and-windows)
 - [Development Invoke Commands](#development-invoke-commands)
   - [Running servers](#running-servers)
   - [Lint checks and auto fixing](#lint-checks-and-auto-fixing)
   - [Static builds](#static-builds)
 
 ## Prerequisites
-#### Docker
--   `Git` [Installing Git](https://gist.github.com/derhuerst/1b15ff4652a867391f03)
--   `Docker` [Install Docker Engine](https://docs.docker.com/engine/install/)
```

Comment:
I notice the docker prerequisites are being removed. Is this an intended change?

Reply:
No